src/main/scala/tech/cryptonomic/conseil/tezos/Tables.scala

```diff
@@ -21,6 +21,7 @@ trait Tables {
     AccountsCheckpoint.schema,
     BakingRights.schema,
     BalanceUpdates.schema,
+    Ballots.schema,
```

Comment:
Are we sure this is wanted and not some unexpected merge error with master?
I think one of the rollup changes included the removal of these tables (ballots and proposals)

#548 for reference

Please check the sql schema and try to regenerate slick's definitions to make sure we're not re-introducing things that shouldn't be there.

Reply:
clumsy me, I forgot to recreate container before generating table, good catch